src/components/Customer/Create/PrivateSwitch.jsx

```diff
@@ -10,23 +10,23 @@ const PrivateSwitch = ({ gId, getSwitch }) => {
     setPrivate(e.target.checked);
     getSwitch(isPrivate);
     axios.put(`/db/maps/${gId}`, { isPrivate: e.target.checked })
```

Comment:
should this have a .then?

Reply:
I don't need any info back from the put other than maybe a verification in the console so should I have a then, then?